### PR TITLE
add livescript support

### PR DESCRIPTION
**Pull request by Announcement**
i'm too lazy to undo the readme stuff so you can deal with that.

### CODE REVIEW

**Comment by tshedor** [comment]
Hey @Announcement I saw your note. Would you please remove the `package.json` and `README` edits? I'll merge after those are cleaned up.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
 # Slim Atom language package
 
-## Looking for maintainer(s)! See [this issue](https://github.com/slim-template/language-slim/issues/25).
```
Comment (auvipy) [inline README.md]: undo readme edits please

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "description": "Slim language package for Atom.",
-  "repository": "https://github.com/slim-template/language-slim",
```
Comment (auvipy) [inline package.json]: and this too.

**Review by auvipy — CHANGES_REQUESTED** [review]
If you don;t have time, to edit, you can allow edits from maintainers, so that we can edit them.

**Pull request closed without merging**